lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -416,6 +474,16 @@ def token_type_from(payment_method)
           'applepay'
         end
       end
+
+      def handle_response(response)
```

Comment:
💬 Comment:

Handle response is not being used anywhere.

Reply:
It's used in the unstore method. I overwrite it to get the response code and create a response properly.